registry/remote/errop/errors.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright The ORAS Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package errop
+
+const (
+	DeleteDanglingReferrerIndex = "delete dangling referrers index"
```

Comment:
1. Should it be something like `delete_referrers_index`?

Reply:
Some example `Op`s from `os` and `net` packages:
- `read`
- `write`
- `accept`
- `chmod`
- `chown`
- `lchown`
- `open`
- `readlink`
- `close`
- `truncate`
- `remove`
- `link`
- `syslink`
- `ReadDir`
- `OpenFile`
- `lstat`
- `parse`
- `seek`
- `readdir`
- `dial`
- `listen`
- `wsasend`
- `file`
- `route`
- `annouce`
- `set`
- `raw-control`
- `raw-read`
- `raw-write`
- `proxyconnect`
- `fork/exec`

Reply:
Camel-casing with first letter capitalized sounds good to me.